lib/nexpose/manage.rb

```diff
@@ -43,6 +43,27 @@ def system_information
       end
     end
 
+    # Obtain the version information for each scan engine.
+    # Includes Product, Content, and Java versions.
+    #
+    def engine_versions
+      info = console_command('version engines')
+      versions = []
+      engines = info.split(/\n\n/)
+      engines.each do |eng|
+        engdata = {}
+        eng.split(/\n/).each do |kv|
+          key, value = kv.split(/:\s*/)
+          key = key.sub('Local Engine  ', '').sub('Remote Engine ', '')
+          if key != 'VERSION INFORMATION'
```

Comment:
i think for this i'd rather see the following:
`next if key == 'VERSION INFORMATION'`
or we could just gsub out `'VERSION INFORMATION'` from the string. 
Since we know it should always be in the string, and we know we don't want it, it might be easier to just gsub it out, and avoid the `if` statement all together. 

But either way thank you for this @Red5d , this is pretty awesome that you took the time to submit the PR!